src/bundle/Resources/views/admin/content_type_group/view.html.twig

```diff
@@ -25,4 +25,11 @@
         page: page,
         routeName: route_name
     })) }}
+
+    {{ render(controller('EzPlatformAdminUiBundle:ContentType:draftList', {
+        contentTypeGroupId: content_type_group.id,
+        page: page,
```

Comment:
TBH it's weird that those lists share the same paginator. What if one list has 2 pages and the other one has 5 pages? If I want to go to 2nd page of drafts, would this call also load 2nd page of published CTs? Not too fluent with Pagerfanta, so maybe I'm missing something.